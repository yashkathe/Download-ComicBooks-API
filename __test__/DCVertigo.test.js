const { getDCComicsVertigo } = require('../index');

test("DC Vertigo", async () => {
    expect(await getDCComicsVertigo(1)).toMatchSnapshot();
});

