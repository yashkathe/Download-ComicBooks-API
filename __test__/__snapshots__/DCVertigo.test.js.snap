// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DC Vertigo 1`] = `
[
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/09/The-Sandman-Presents-Thessaly-Witch-for-Hire-TPB-2005.jpg?fit=400%2C615&ssl=1",
    "description": "The powerful and ancient witch Thessaly’s attempts to live a normal life are interrupted when a lovesick ghost named Fetch tries to win her heart.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/WDvW1zpBxJjWGc7rHE3aBEQR9Nptxns8maYarFZ5GLF/TzUIy9w6nQHCtr4N/wr5vk3hmjunEK1anIe9F4JYbi0VhouINiknpte9COCuLET+Naq5APe1+XNgfEZeK7R3ZDjJRsekcx+Kb6iNDOCAk/rLMFVeWhmZnfLClXuMxXF6j/kQ2Zt6Lv4/51M8aA1oMhCsIAw/92Qg49co5wCiCbpRSab+FF8JT1IBOlSD66jluxT52gUb8+uunSMiIhtg:OGpzSpjijvvMs1+67dIhbQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php/KuJKMxsCl+cwuULAxmGpHtrJO3lFwMWShlrp10dhQ8TmGC2w6vWBtKPKuILaJZfPAxO5bOu+giHEN/EKKGa8C9HX4eagw/9ix/jeagzGEJimqw6r2mOqnrt2wVANXm+WMEQym/kFAnnZArRf5/YNXnVTD2l09R0Dylpz5n3/PXXoZ5RAxYOw2mP0cxQgmWHMIONx205RLrh4RWJQi2F6M5z882qEstMBl7VazSoqBAw=:Ibj/a+azgkOYe96OvgiN6w==",
      "MEGA": "https://getcomics.info/links.php/PNFwODFiRxeny8MQasvbVGMA2JMUjuca2UuSeBCWYByckEJ8FrB/Ga45L1MqKJsULO5RnJ6N7OUcfVvIMoaHyFP9wlAk7XKcm3VaC9Im1qc=:ipMi69VrPquKK0WInedkNg==",
      "UFILE": "https://getcomics.ufile.io/2g5jzsar",
      "ZIPPYSHARE": "https://www13.zippyshare.com/v/vkmeg3nh/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "326 MB",
      "Year": "2005",
    },
    "title": "The Sandman Presents – Thessaly, Witch for Hire (TPB) (2005)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/09/The-Sandman-Presents-The-Taller-Tales-TPB-2003.jpg?fit=400%2C615&ssl=1",
    "description": "Meet Merv Pumpkinhead – Agent of Mystery, called in when good dreams go bad. Learn everything you ever wanted to know about dreams, and answer all those nagging queries about sex and death.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/OYj02wS5Z+2nvtDU81cFp+FO5N71arb8QlZB60Qz/7gW12u3uuwbWufEVRijRoyHCQOnWtcupYKwAPSUh/0u9sV/jc1H0Rc73l5L9ahh/ByA9NzAhoK97m97yHSrb+QIKM6+Ql+RsE1Cc4+CKOLbbro7EoronHi1j5bYnn7VMMIrMe6RD6qdKGwuib3adRr98aXRidg2h8Jgv0y2eQQupwBS/IJmI0+ygWuLkaD+P4E=:RRCzdwOy/9vcdGPC69KRqw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/Tk2ozpNYhNKY9o48mc7EQ49kKfVgX2Y7Bi6y5D4qf/21Tlt3HNfoixSKnmoVK1m5/1BT216X862LL06ys7pk5jT1/PZXkBOX1Yc9oZSzds0MA/3lQahTrgJoIjzZyBQXzxnVPSv6cwanwu/7XtrLEFx6cIstdi2OMxm1KIyqkFg5tb3BFSsK+WnFqj2uJrzR0YM5Wi9Q63muQHPuMLa1Cg==:1S8Jc14wVsvdVqRhbh1YPg==",
      "MEGA": "https://getcomics.info/links.php/2rKlu3WpPRpKhwob/zA55nANELzy3ZDOP/tO6nQQwR7Xxz50Zrb+n6OHYYV6D7mm7dDMWudSG0Zf96H3ZaxKVlbgHGAJ5sbHCSR996f1O/0=:1CLXF0C7GrgZLHPfQSnfig==",
      "UFILE": "https://getcomics.ufile.io/aime5gjm",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "757 MB",
      "Year": "2003",
    },
    "title": "The Sandman Presents – The Taller Tales (TPB) (2003)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/09/Destiny-A-Chronicle-of-Deaths-Foretold-TPB-2000.jpg?fit=400%2C615&ssl=1",
    "description": "In the plague-riddled years of the early twenty-first century, the inhabitants of a small village live in fear until an enigmatic stranger arrives, offering a prophetic glimpse at the Book of Destiny, and the hope of salvation.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/B1PH1iov/Aj+fFJX1yp8w/mgvZ58Cr1/KxnTpoFEa76YxzEK4pw3bocSgn3aq+AD0DcKVRa4sSZ4udy1q8skfZkpznwLXMGznK2t+9yvJxZj5iUMiLcaszxJW6pH0ysg8hWdPEgnc5Y3EeCgViV8NF8lATw1eyzOA+q2bqddI/I0Yo4oXVxbSkwS20gm+8QvI77SARIK7KVk2eGrta/a+Z7/phS7WQhvOm1dMDw/3rg=:hRJcxr8RQdB1/LyNw6TYFA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/puJNvCCZxZSBG0aJxV9MX/M6lAF3hWgvAwlYUciz0tWmlcpHT065Mwe7HlLTb1n/SCwpUED/Cygf72fFTBj0i1OSDaW1k9+AHzZXAWeTkz88BeU8d8Dl9Xoi45Thtk/cHwtZL2Qqsv8dfK56UpFi5xiYGZF6sY0efCr5A1yMOc5FARYNtSNGK7LToVlgEll++DOm20CPYenbzEXP2ULAvG24f0gyDDfMwJ2GS9D4SCA=:tUVlTOqyaZNf01uxm33p9A==",
      "MEGA": "https://getcomics.info/links.php//XFt4Tceg2+7X5HobbAew4VekwmtpxHBGYk8hasOqPn16GLFmITc8OEOncrRDJvVC7LSi5x6Knumk/sDxVb3DGWbmhWEg5wTYbCBnFudY/g=:+xDwQsUvgf5oa8Wab67hTA==",
      "UFILE": "https://getcomics.ufile.io/72b8janb",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "784 MB",
      "Year": "2000",
    },
    "title": "Destiny – A Chronicle of Deaths Foretold (TPB) (2000)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/09/Death-The-Time-of-Your-Life-TPB-1997.jpg?fit=400%2C615&ssl=1",
    "description": "Written by Neil Gaiman; Art by Chris Bachalo, Mark Buckingham, and others Death incarnate, as defined by master storyteller Neil Gaiman (THE SANDMAN), is a genuinely likeable young girl with a fondness for ankhs who truly cares about people. It’s small wonder then that when a rising star of the music world wrestles with revealing her true sexual orientation just as her lover is lured into the realm of Death that Death herself should make an appearance. A practical, honest, and intelligent story that illuminates “the miracle of death.”",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/v9DfpaMQlV1q6rOjv9vZFd1bbnJP+M2wm8KJWIOj4XZNlGgySx8lgoYFKUijgmQHtUcfniRTmHETv7cH3S0/52vRf1U9KSbExa7xfTMsN3IW1shsCla/fCNEsK8sleeI12XbIUCdlERouJSNiCSNEB3ObN5NPDuzT8HmYzv17LkeR6PKofZ8boRhcM8b/40MmMH9UaCSab/Ei/fTnMwKHQ==:kHV22xeITvxeWJk8iqX9Eg==",
      "MEDIAFIRE": "https://getcomics.info/links.php/EJfJ5gWI2cBN2quH4DwFs2MwtTZuvKAFhAPG+k61VLuikSMW8eS+WcHTFDjsOeMB9KRYATUqUjjLGbUqGGrl8gZk99EJchvgWoVfkQmS5oDJgn6QCkuWeoMfbenYYTJI2DqYgv6Wq18Pw4LXMw/8Xc9Q7aKQz9bcP4Tjpujx5yktUCD5Xz306qHiewAxGNNfRnad9tmQW/hKxjywWSaTkw==:uuT0M1CllN4I2E6IoVMGrA==",
      "MEGA": "https://getcomics.info/links.php/VbN268eMJ4cwOEK2K0WiX+zF6sllLr03zgsfkn9wc7gJXtapzks0bicwCwLzB6kbEzPIAf2ABzqZBjEF06vQhPb5Kpj3dTvVGb1J5p8rQto=:8aJPXwy7VaD/wZEOAA/m3w==",
      "UFILE": "https://getcomics.ufile.io/pq0zaqzd",
      "ZIPPYSHARE": "https://www5.zippyshare.com/v/B5eA53e8/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "335 MB",
      "Year": "1997",
    },
    "title": "Death – The Time of Your Life (TPB) (1997)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/09/Death-The-High-Cost-of-Living-TPB-1994.jpg?fit=400%2C615&ssl=1",
    "description": "From the pages of Neil Gaiman’s SANDMAN comes the young, pale, perky, and genuinely likable Death. One day in every century, Death walks the Earth to better understand those to whom she will be the final visitor. Today is that day. As a young mortal girl named Didi, Death befriends a teenager and helps a 250-year old homeless woman find her missing heart. What follows is a sincere musing on love, life and (of course) death.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/W84WTi7hB1omkeAoBQ/Z+EYQUgCh+BQLkivbS7PjGu3/ckIr2ZRHaBsL65EjSuW47KYBzfdkUG2FnrQtrjzObKVlk6fuVXzwhLN2rTejFbDUpdIc1igy9BejDUqd43I+QT2bFe2722XCDU7ybGN24jU6krHfVUGdVw5SA6GeAeQjoMq5kAUAYNSszKeT7RifF+a4kUxhKQpJOaV+bO71ng==:BG0T3gLV+pQLytChGfN7Cw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/Ubw6xiWY6ulw3bDwXyJO3bps7EjRLZbrN+IyGAHCxLyTcAIEZ8K5qsLU0VrlkqM2aYWLVUXNzaYEBSYSn51VzSVCmZapxHuMmRaHPizwe/uRiqjcdCVYLXrCV9nrY4mixn2PCqMlpxW35oTMeKgYhWmXzNh7xY2B09AbYxj9/mQt8Dkz5NFi7wSubiTPKlT83PJOEEya+uMZOJQVkYdrKg==:FyqTs+LV7PMeJuA8Whl0pg==",
      "MEGA": "https://getcomics.info/links.php/VZ+NDpLizp/tVW4pSFBmBLc9wSL+ZmL7Mfw4KGkn5NOAImHYLPmVwRJCAjtQKjsMEvL/9Ge8Bvm35UMONZzmfJO6OYGIOcTsJ6Cb2T71YtY=:SREQlh0Qug8o3WuoHv/h9w==",
      "UFILE": "https://getcomics.ufile.io/mmz9acjv",
      "ZIPPYSHARE": "https://www5.zippyshare.com/v/3HtkkzeL/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "291 MB",
      "Year": "1994",
    },
    "title": "Death – The High Cost of Living (TPB) (1994)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/08/Destiny-A-Chronicle-of-Deaths-Foretold-1-3-1997.jpg?fit=400%2C614&ssl=1",
    "description": "Beginning a 3-issue miniseries delving into the past and future of Destiny of the Endless, with painted covers and framing sequences by Kent Williams. In the year 2009, a small town holds out against a devastating pandemic, its remaining inhabitants desperately hoping to survive the next outbreak of plague. Then a mysterious stranger arrives from nowhere, selling a page that he claims is not only taken from the Book of Destiny, but also foretells the four great plagues of humankind, including where the current one will strike next. To convince them of its worth, he offers to read from the page, revealing Destiny’s role–as well as his own–in the previous three plagues, starting with the epidemic that struck the Byzantine empire.",
    "downloadLinks": {},
    "information": {
      "ImageFormat": "JPG",
      "Size": "599 MB",
      "Year": "1997",
    },
    "title": "Destiny – A Chronicle of Deaths Foretold #1 – 3 (1997)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/08/Crossing-Midnight-Vol.-1-3-2007-2008-Fan-Made-TPB.jpg?fit=400%2C599&ssl=1",
    "description": "Set in present-day Nagasaki, Japan, a fantasy/horror series begins when two extraordinary twins are born–one before and one after midnight– and one of them has access to a world filled with supernatural beings who can enter, and impact, the world as the commonly know it.",
    "downloadLinks": {},
    "information": {
      "ImageFormat": "JPG",
      "Size": "496 MB",
      "Year": "2007-2008",
    },
    "title": "Crossing Midnight Vol. 1 – 3 (2007-2008) (Fan-Made TPB)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/07/Astro-City-Metrobook-Vol.-1-TPB-2022.jpg?fit=400%2C615&ssl=1",
    "description": "A hero dreams of flight. An alien spy prepares for invasion. A young man is mentored by a hero with dark secrets. A street criminal discovers a hero’s identity. And much, much more. Step into a world of heroes and see them from a whole different perspective. These are the multiple-award-winning stories that began the epic series and changed how we think about superheroes.Collects KURT BUSIEK’S ASTRO CITY, VOL. 1 (#1-6) & VOL. 2 (#1-12, #½)",
    "downloadLinks": {
      "MEDIAFIRE": "https://getcomics.info/links.php/C1prnUakzhSAo6MzqVi8I5WpYsSV5MAUksoWcriHLfCzXXyTsHQ7SDNyBUvwTl5UODwUGlrS+3A88cBQHp+BBr+V37lLI3jxKEN1y9zWmW/28MD2j239WfyHQdVv6qsfjckHYUEPKYWDT4mIIb6DdETZUMJnW7zZTfKaf9BQVR4jdJG1Bt2saslJur9N0z01jGrAGXxPCDgQcUsga6uFWw==:jwF1bt6OW9BBOU1fMirvyw==",
      "MEGA": "https://getcomics.info/links.php/1NKKEO3I3RweIxZn1Rxh35Z2ILhUO20B1+6pVOTcM9rJOj1e8NecMatUxcvRVztUp6KOQEW+VrnBJH1s3JkbNxwdiUmG37XoyRF0beFPaX8=:38VhUhPcukisaSfPe4yAhg==",
      "UFILE": "https://getcomics.ufile.io/uq43voai",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "913 MB",
      "Year": "2022",
    },
    "title": "Astro City Metrobook Vol. 1 (TPB) (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/06/The-Names-of-Magic-1-5-2001.jpg?fit=400%2C615&ssl=1",
    "description": "Tim Hunter is back, and he has finally accepted the role he plays in the grander scheme of things, as he learns to recapture his magic.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/uD2juV3doGmXFBk2ETm9XXFGeAzF9uYVkTQotn9tz8GKg6JCnNYi/GMoa7LS7t1I8ekiDPGW78iQ7ZRGuSvOkBTJheNzuORwTrnPDJ4u72UvV2fLnNYOmRzwxXerBaiyIq7ICSgjjE1aMDHboByyB/qBImlh4qUQtLn8CVeesEI5gIk306W7iEaItH48clSauooc/zjbgy+dADVt/spNig==:DCqiDTrvoEG6O0uBT8PeMQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php/QOoeq3M1oWcPhk92QU5MIbdhXzCUB7J4kaAs/2q5oIxc7xz5EcjAJ2umnY8GwBHWlbrvALTC+ry7dnEWzbObekgXsYXeb5Vl/bFECQ0tp1avJGPnUy5ebr/MJtFEwK64yBJi5hxVMKkuYPNxlibvyEbGleEQtxG/BF3iB2AVwnfQzkBN+8qJoXLHWkrRKFVpsLIyTo9KXSrGP9foVDNwKQ==:EKKcs5RtC1iuSChtZ1VoVw==",
      "MEGA": "https://getcomics.info/links.php/cWYNSUGOH3GnPfQjmRFsntYoh1V2tlJypFVs+MFeN1kJO61ufemZ8z9EP1tR/vbbo0R9H/YIzjldEKYP+/dZC509vocCf4HN/1bCfY5/4Z0=:5+mRhXk0shZbtPDU68mdcQ==",
      "UFILE": "https://getcomics.ufile.io/5t3946jg",
      "ZIPPYSHARE": "https://www21.zippyshare.com/v/1jxYpZac/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "219 MB",
      "Year": "2001",
    },
    "title": "The Names of Magic #1 – 5 (2001)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/06/Death-The-Time-of-Your-Life-1-3-1996.jpg?fit=400%2C618&ssl=1",
    "description": "A Sandman spin-off from writer Neil Gaiman with art from Chris Bachalo . This story brings back characters Donna Cavanagh (who prefers to go by the name of Foxglove) and her lesbian partner Hazel McNamara from a previous Sandman series of stories, A Game of You .",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/Dd/9TZ8BBmCBiMysITJmBQLZJOHZfMqh4D5O8YxE7b/KlxJlsl08+SaMmonZeRfvYZWsi5JhPeYiFxAfEfAtIFDUEfw0xHlqpTuFxW/uhfm1v/HsLo6KdxBd1Yac+4jDo87CCpDNk9hX+1tG1KhdL6C6Cegz+Ka/ZDWukJ/wFmF3YEiUFfRzksiQEw2vypKUwQwyMtKQhu1T0FOZ3+mtCR3miB0wU2n9iF4tXwSvh9U=:mhLl4aAjmrQPD8ZeB42ZkA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/9ufJ9U/XrD8kuz4isqdv90lAbQ/F6xEwSZOCD82Z28koqxaH6q1e77zchShL1Oaitcda0Ze6sqskS23TFHjKEhN9ieyxznfP7jB91EPTvxhwnc1vO2zYYbTQcioPWfcpoEiZBaoZo4XpgETYEGGz8i/XzLD0hYTh82qRX7bcS+zQfl/T0aFYawzGeWSgflwBU86h4HGJaD+2/R2g/jluPw==:A4PwD8CjiSJ6xF1N4ohMSw==",
      "MEGA": "https://getcomics.info/links.php/HuxFemYnkynNCKa+EEqatHO3FUGfsbIeBju7scKJ4bn1niSrQMNoGVHLCIujyeomzOqg6wn17YBG4/zHT6n7RbZwclPa+ezdSdz3ei3MMmk=:UmEBY85chXmcBgaJorHQ1w==",
      "UFILE": "https://getcomics.ufile.io/frhzc6a3",
      "ZIPPYSHARE": "https://www21.zippyshare.com/v/ijZc1xFl/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "140 MB",
      "Year": "1996",
    },
    "title": "Death – The Time of Your Life #1 – 3 (1996)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/06/Death-The-High-Cost-of-Living-1-3-1993.jpg?fit=400%2C614&ssl=1",
    "description": "Three issue miniseries chronicling Death’s day as a mortal.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/fQk21YOr8Rhq76b0mC6yFbGhopj7ts7E6ZX+83jlzcV1eKkwUXgnNL5R9zAHHRZU7mJBjCYpUGmHaS4qt0RFoOFyZ6/D5W75y7bK2/e7An0fAtv2GSttc9Xsr4zZO0NF8JKnRnXmVul3FmoWzDc0u5UpYsLSJfAaYoir6ZW75Qvv8uiRkjBtuUO2MLI4Xolp:kVuLbevW5dSHoo5ED274cQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php/l6nDIIgV8rEBqEN87PcbDzvUqxP59V0OOeZJS+XOrd2AY/R8aaU7dczPIWJTwv7UvdP198YBNmyc9ayVeifFDOBducEgrSECAFxXw0Qcjuc4NALmFhaj9nckGT3n3sTscNg3VnY3bPd9x27jlDP0EM+Z8U3oTG1uHk7UHH9yWzLIvr0EI1h8BpjWiRm+P8jh:am44aJ73y5GvFDTDAnq15A==",
      "MEGA": "https://getcomics.info/links.php/Wx9WPHYG80uRp2D3GZFavmgfiR50G2J3aMP9VM4/kTBTzWxGeXNGYczk9YE2mzJxgaiY3Hlxo4k7HfCAQGrggpxkhgD1UBiun8BxdcMgvss=:KcYNkT5CNlkK2HjrvTQR4A==",
      "UFILE": "https://getcomics.ufile.io/67k8nxf6",
      "ZIPPYSHARE": "https://www21.zippyshare.com/v/PV48utez/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "83 MB",
      "Year": "1993",
    },
    "title": "Death – The High Cost of Living #1 – 3 (1993)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2019/11/Vimanarama-2005.jpg?fit=400%2C614&ssl=1",
    "description": "From the fertile mind and nimble fingers of writer Grant Morrison and artist Philip Bond comes the latest in multiculturalist spectacle! Color, song and cosmic violence collide in VIMANARAMA, collecting the 3-issue Vertigo miniseries that fuses East and West together into a transcendental delight of romance and danger. Feel the earth tremble as 19-year-old grocer’s son Ali and his beautiful arranged bride Sofia accidentally unleash an ancient race of unimaginable evil from beneath Bradford, England, then summon the equally powerful Prince Ben Rama and his Ultrahadeen to do battle against them.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/CMG+a/B30JOWJPGT5eOA9A12nRA5RjBRsZVtWYYzruqLX0YqAT02SsmJfVcnpvaH5HgmDKs9FkOgv1y2OCB/jbyhTJwLzZItSGB1MsNdWFr8M47q2zHtlD2J/GwaT7cvgw3aFVkeYRjsW74iOCTBpcA2VaN4UVeyD+eXR6yNzsE=:f04aOWN/Zs1FN8/lBiZDMw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/auoVXKNnklLsW0YjOXk/R89vdyuSIQQBuqxfzb6pFzdddi8bwHnS2Q/ZJ13BMNRE3yRcbe9fx1O3pKiynpNAyLu0PD65nrjdzYj+VNQI8h/BvtHHdlGTt4M7oorKJ6Z9b+Jgv2lv1dTECQnJ6GJq5ElYtaXxUfOYvFOUzhHNgtH0bp+LRB7CoDfRBCDAiiKM:vtjMwwRpP7tQejrp3YzOBQ==",
      "MEGA": "https://getcomics.info/links.php/Yrc9lWN7LM+o8Lxww5fwdoC8zbJelmRp/xwolqwrXRQKGk565MeV8kS5989Ml3XgYIpO5GtPAmvJxq6PFxiQjY2l12cSW8G+NPhWeXUeQ3g=:+kl+29yyd3+lv9rJM7u8wg==",
      "UFILE": "https://getcomics.ufile.io/4gqrejlb",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "108 MB",
      "Year": "2005",
    },
    "title": "Vimanarama (2005)",
  },
]
`;
